src/video-player.js

```diff
@@ -510,6 +510,11 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       }
     });
 
+    this.videojs.on('canplaythrough', () => {
```

Comment:
@yaniv-li should this replace the fix done in the previous pr or is it an addition?
Shouldn't we just make sure the retry() function clears the timeout when needed instead of listening for events?

Reply:
@maoznir 
I think it can replace the fix from the previous PR that said they are not conflicting.
I think listening to events is better than trying to figure out the state of the file after 55 seconds.